mobile-app/app/screens/AppNavigator/screens/Portfolio/components/ActionButtons.tsx

```diff
@@ -41,6 +52,17 @@ export function ActionButtons(): JSX.Element {
         showsHorizontalScrollIndicator={false}
         horizontal
       >
+        {hasFetchedToken &&
```

Comment:
Create a new const for this, so the condition here will be simplified. (Easier to read)

const hasDFIBalance = condition...
{ hasDFIBalance && <element...> }